src/keri/db/dbing.py

```diff
@@ -408,15 +412,25 @@ def reopen(self, readonly=False, **kwa):
             fext (str): File extension when .filed
             readonly (bool): True means open database in readonly mode
                                 False means open database in read/write mode
+            cf (Configer): optional Configer to configure the opened LMDB database via kwa
         """
         exists = self.exists(name=self.name, base=self.base)
         opened = super(LMDBer, self).reopen(**kwa)
         if readonly is not None:
             self.readonly = readonly
 
+        cf = cf if cf is not None else self.cf
+        config = cf.get() if cf is not None else None
+        lmdberConf = config.get(LMDBer.ConfigKey) if config is not None else None
+        dbconf = config.get(self.ConfigKey, None) if config is not None else None
+        if dbconf is not None:
+            self.mapSize = dbconf.get("mapSize", 104_857_600)  # 10MB default
+        elif config and lmdberConf: # fall back to LMDBer size if present
+            self.mapSize = lmdberConf.get("mapSize", 104_857_600)  # 10MB default
+
```

Comment:
We discussed in an implementers call that the dependency here is not correct.  Lets discuss at the next call.